Common React Native components

skatshang opened the pull request:
This pull request added React_Native_Component.md, including the following changes:
1. 20 React Native components with explanations
2. Commonly used props of each components, with their explanations and accepted type

Future updates to consider:
1. Styling
2. Add link to official documentaion

Please review the changes and provide your feedback. Thanks!

kevintan999 commented:
Hey @skatshang, would you mind adding some of the official documentation that React Native has? i think some hyperlinks could work really well and point to more information.

kevintan999 commented:
And maybe add some type of introduction to introduce the reader on what this article is trying to teach?

skatshang commented:
> Hey @skatshang, would you mind adding some of the official documentation that React Native has? i think some hyperlinks could work really well and point to more information.

skatshang commented:
> Hey @skatshang, would you mind adding some of the official documentation that React Native has? i think some hyperlinks could work really well and point to more information.

Thanks for your advice! The links to the offical documentation have been added. They also helped me realize that some components I listed have been deprecated and should be removed from my introduction.

skatshang commented:
> And maybe add some type of introduction to introduce the reader on what this article is trying to teach?

A summary has been added.

kevintan999 commented on Topics/Tech_Stacks/React_Native_Component.md:
```diff
@@ -0,0 +1,295 @@
+
+## Summary
+
+This guide provides an overview of commonly-used React Native components for mobile app development.
```
Summary looks good!

skatshang replied:
Also added anchor links in Table of Contents under Summary, to help readers quickly jump between the sections.

kevintan999 commented:
@skatshang maybe for some entries you can add a few code examples? that could help users to identify sample usage. Or add some customization tips to provide more insight.

skatshang commented:
For the summary part, I also added anchor links for readers to quickly 

> @skatshang maybe add your page as an extension in Learning React Native. Or add some customization tips to provide more insight.

just added my document as a section in Learning React Native page. thanks!

kevintan999 commented:
The new changes are good, thanks for the good work! 

looks good to merge!

kevintan999 commented:
looks good to fix!

skatshang commented:
> the links are not rendered on the website properly. please change the format.

Noted. will start a new PR to fix it
